fendermint/app/src/app.rs

```diff
@@ -136,6 +139,8 @@ where
     builtin_actors_bundle: PathBuf,
     /// Path to the custom actor WASM bundle.
     custom_actors_bundle: PathBuf,
+    /// Block height where we should gracefully stop the node
+    halt_height: i64,
```

Comment:
The `app` crate has a `BlockHeight` alias. Unfortunately there is no common place for it, so it's declared in multiple crates, but it would be good to try to use it where it's available. Unlike `ChainEpoch` it's a `u64`.